common/config/sizeauditor/sizeauditor.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sizeTolerance": "1024",
```

Comment:
Do you want this numeric type to be written as a string?

Reply:
I have changed this to `long`. Also the latest commit will not contain this field, since it has a default value of `1024` set. It should show up in the `https://github.com/microsoft/json-schemas` once I make the change.